docs/consume-packages/Package-Source-Mapping.md

```diff
@@ -100,7 +100,7 @@ To learn more about how package installation works, see [the conceptual document
 
 ### Get started
 
-To fully onboard your repository you may take the following steps:
+To fully onboard your repository you can do it 2 ways either manually or automated [NuGet.PackageSourceMapper tool](https://www.nuget.org/packages/NuGet.PackageSourceMapper/0.1.3-preview.22218.1#readme-body-tab), for manual onboarding you may take the following steps.

```

Comment:
How about a note here?

https://docs.microsoft.com/en-us/contribute/markdown-reference#alerts-note-tip-important-caution-warning

Also let's just link to the package, not the version nor readme. that way we dont have to update the docs with versions.

Reply:
I improved the text based on feedback from you and Nikolche, please check again.